config/locales/en.yml

```diff
@@ -25,11 +25,33 @@ en:
       strip_insignificant_zeros: false
       unit: "$"
 
-  titles:
-    application: Budgetapp
+  application:
+    show:
+      button: "New Account"
+      new_account: "To sign up for a new account click the 'New Account' button"
+      welcome: "Welcome to Budget App"
+
+  users:
+    new:
+      name: "Name"
+      email: "Email"
+      password: "Password"
+      submit: "Sign up"
+      title: "Please fill in the form to sign up"
+    edit:
+      name: "Name"
+      delete: "Delete Account"
+      email: "Email"
+      submit: "Edit profile"
+      title: "Please fill in the form to edit your profile"
+    show:
+      output: "Your email address is: "
+      welcome: "Welcome"
 
   activerecord:
     errors:
       messages:
         blank: "can't be blank"
         taken: "already taken"
+
+  # submit: "Sign up"
```

Comment:
Remove comment??

Reply:
Woops, thanks for picking this up, haven't noticed it there!